packages/cli/src/controllers/cta.controller.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+import { Authorized, Get, RestController } from '@/decorators';
+import { AuthenticatedRequest } from '@/requests';
+import { CtaService } from '@/services/cta.service';
+
+/**
+ * Controller for Call to Action (CTA) endpoints. CTAs are certain
+ * notifications/messages that are shown to users in the UI.
+ */
+@Authorized()
+@RestController('/cta')
+export class CtaController {
+	constructor(private readonly ctaService: CtaService) {}
+
+	@Get('/')
+	async getCta(req: AuthenticatedRequest, res: express.Response) {
```

Comment:
We might want to make this endpoint more specific? If we add more CTAs, it might get confusing that `/cta` is this activation CTA but `/cta/something-else` is another.

Reply:
Good point. I think it would make sense to make endpoint only for this specific flag. Will change